R/google_drive.R

```diff
@@ -0,0 +1,189 @@
+#' API to get a list of files in Google Drive
+#' @export
+listItemsInGoogleDrive <- function(teamDriveId = "", path = NULL, type =  c("csv", "tsv", "txt", "folder", "xls", "xlsx")){
+  if(!requireNamespace("googledrive")){stop("package googledrive must be installed.")}
+  token = getGoogleTokenForDrive()
+  googledrive::drive_set_token(token)
+  if (!is.null(path)) {
+    path = googledrive::as_id(path)
+  }
+  #if team id is provided search documents within the team.
+  if(teamDriveId != "" && !is.null(teamDriveId)) {
```

Comment:
Check if we can pass NULL teamDriveId and consolidate code for those 2 cases.

Reply:
Changed to pass NULL as suggested.